customizations/ncv4.sh

```diff
@@ -12,6 +12,7 @@ ln -sf /opt/microsoft/ncv4-graph.xml /opt/microsoft/ncv4/graph.xml
 bash -c "cat > /etc/nccl.conf" <<'EOF'
 NCCL_TOPO_FILE=/opt/microsoft/ncv4/topo.xml
 NCCL_GRAPH_FILE=/opt/microsoft/ncv4/graph.xml
+NCCL_IGNORE_CPU_AFFINITY=1
```

Comment:
Is this intended?
The commit message says "Modify **ND** sku customization scripts to add "NCCL_IGNORE_CPU_AFFINITY=1" to the nccl.conf file"

Reply:
Thanks Jie. My description should have included the NCv4.